presto-docs/src/main/sphinx/release/release-344.md

```diff
@@ -0,0 +1,64 @@
+# Release 344 (XX Oct 2020)
+
+## General Changes
+
+* Add {func}`murmur3` function. ({issue}`5054`)
+* Add {func}`from_unixtime_nanos` function. ({issue}`5046`)
+* Add [T-Digest](/functions/tdigest) type and functions. ({issue}`5158`)
+* Improve performance and latency of queries leveraging dynamic filters. ({issue}`5081`)
+* Allow to configure (using ``dynamic-filtering.service-thread-count`` feature toggle) number of
+  threads used for processing dynamic filters on coordinator. ({issue}`5341`)
+* Extend environment variables substitution (`${ENV:variable-name}`) to work also when the substituted text
+  is part of the configuration value. ({issue}`4345`)  
+* Remove `fast-inequality-joins` configuration property. The fast inequality joins remain enabled. ({issue}`5375`)
+* Report dynamic filters collection duration. ({issue}`5434`)  
+* Improve query performance by propagating more detailed dynamic filters to coordinator. ({issue}`5340`)  
+* Improve performance and accuracy of {func}`approx_percentile`. ({issue}`5158`)
+* Improve performance of cross join queries. ({issue}`5276`)
```

Comment:
all of them? or only some cases?

Reply:
let's indicate "some cases" somehow, without going into details
oitherwise people may expect their particular cross joins to run faster and may be disappointed